Makefile

```diff
@@ -77,6 +83,39 @@ cuda-debug:
 endif
 
 ifdef ALPAKA_BASE
+# Alpaka/cupla implementation, with the CUDA GPU async backend
+#Compile-time device choice implementation
+alpakaCT: main-alpaka-ser main-alpaka-tbb main-alpaka-gpu
+	@echo -e $(GREEN)ALPAKA, compile-time version, targets built $(RESET)
+
+main-alpaka-ser: main_alpaka.cc rawtodigi_alpaka.cc
+	$(CXX) $(CXX_FLAGS) -DALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED -DDIGI_ALPAKA -DALPAKA_ARCHITECTURE=CPU_SERIAL -I$(ALPAKA_BASE)/include/alpaka/ $(CUPLA_FLAGS) -o $@ $^
```

Comment:
Is `-I$(CUPLA_BASE)/include` needed here? (I'm wondering if the `$(CUPLA_FLAGS)` could be replaced with `$(ALPAKA_FLAGS)`)

`$(ALPAKA_FLAGS)` contains `-I$(ALPAKA_BASE)/include`, and this line adds `-I$(ALPAKA_BASE)/include/alpaka/`. Are both really needed?